lib/posix-process/Config.uk

```diff
@@ -14,6 +14,8 @@ if LIBPOSIX_PROCESS_PIDS
 		int "Largest PID"
 		range 1 32768
 		default 32
+		help 
+				This value represents the maximum number of pids that can be created
```

Comment:
By that help, string I meant that, including the start string, you can create that many threads. With the new changes it is indeed the maximum number that can be created. Because it is easier to understand, I am inclined to completely erase the help message or change it to what you commented. What is your opinion?

Reply:
I think because the option says `Largest PID`, we can live without the help text.